userspace/libsinsp/threadinfo.cpp

```diff
@@ -1721,9 +1714,11 @@ void sinsp_thread_manager::remove_thread(int64_t tid)
 	/* This should never happen but just to be sure. */
 	if(thread_to_remove == nullptr)
 	{
-#ifdef GATHER_INTERNAL_STATS
-		m_failed_lookups->increment();
-#endif
+		// Extra m_inspector nullptr check
+		if (m_inspector != nullptr && m_inspector->m_sinsp_stats_v2)
```

Comment:
Yes the problem child condition: asan will fail if we don't check here because of `TEST(sinsp_thread_manager, remove_non_existing_thread)`, change the unit test?